src/components/modals/InfoModal.tsx

```diff
@@ -74,7 +74,7 @@ export const InfoModal = ({ isOpen, handleClose }: Props) => {
         </p>
 
         <p className="local-font text-left text-sm text-gray-500 dark:text-gray-300 pb-2">
-          ヒントモードは、下の五十音図を使用します。
+          ヒントモードでは、下の五十音図が使われます。
```

Comment:
The original version reads like "the hint mode" acts actively to use the chart.  In this case, we often use passive case.